src/test/java/ua/com/juja/microservices/integration/KeeperSlackBotIntegrationTest.java

```diff
@@ -30,14 +53,138 @@
     private MockMvc mvc;
     private MockRestServiceServer mockServer;
 
+    @Value("${keepers.baseURL}")
+    private String urlBase;
+    @Value("${endpoint.addKeeper}")
+    private String urlAddKeeper;
+
+    @Value("${user.baseURL}")
+    private String urlBaseUser;
+    @Value("${endpoint.userSearch}")
+    private String urlGetUser;
+
+    private UserDTO userFrom = new UserDTO("f2034f11-561a-4e01-bfcf-ec615c1ba61a", "@from-user");
+    private UserDTO user1 = new UserDTO("f2034f22-562b-4e02-bfcf-ec615c1ba62b", "@slack1");
+
     @Before
     public void setup() {
         mockServer = MockRestServiceServer.bindTo(restTemplate).build();
     }
 
     @Test
-    public void fakeTest(){
-        //delete it
-        assertEquals(1,1);
+    public void onReceiveSlashCommandKeeperAddReturnOkRichMessage() throws Exception {
+        final String KEEPER_ADD_COMMAND_TEXT = "@slack1 teems";
+        final List<UserDTO> usersInCommand = Arrays.asList(new UserDTO[]{user1, userFrom});
+        mockSuccessUsersService(usersInCommand);
+
+        final String EXPECTED_REQUEST_TO_KEEPERS = "{" +
+                "\"from\":\"f2034f11-561a-4e01-bfcf-ec615c1ba61a\"," +
+                "\"uuid\":\"f2034f22-562b-4e02-bfcf-ec615c1ba62b\"," +
+                "\"direction\":\"teems\"" +
+                "}";
+
+        final String EXPECTED_RESPONSE_FROM_KEEPERS= "[\"1000\"]";
+
+        mockSuccessKeepersService(urlBase + urlAddKeeper, EXPECTED_REQUEST_TO_KEEPERS,
+                EXPECTED_RESPONSE_FROM_KEEPERS);
+
+        final String EXPECTED_RESPONSE_TO_SLACK = "Thanks, we added a new Keeper 1000 in direction {teems}";
```

Comment:
you must return message to slack like this "Thanks, we added a new Keeper @slackName in direction {teems}". with slackName!